### PR TITLE
Certificate template-GCI

### PR DESCRIPTION
**Pull request by ant159**
Template for certificates
c3
c4

### CODE REVIEW

File: src/app/options/options.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div >
+  <h1><font size="5">Options:</font></h1>
+  <ul>
+<li><a routerLink="/certificate2"><font size="4">Style1</font></a></li>
+<li><a routerLink="/certificate"><font size="4">Style2</font></a><li></ul>
+  </div>
```
Comment (nikitadhiman) [inline src/app/options/options.component.html]: this option to select from templates would be provided on organization's dashboard. please remove it from here

**Comment by nikitadhiman** [comment]
Design is great btw

File: src/app/options/options.component.html (hunk 1 above)
Comment (ant159) [reply src/app/options/options.component.html]: So I just have to delete the option component  and put the designs on the same page ?

**Comment by ant159** [comment]
Thank you!

**Comment by ant159** [comment]
@nikitadhiman  Is it okay now?

**Comment by nikitadhiman** [comment]
Last step, squash your commits

**Comment by ant159** [comment]
> Last step, squash your commits

Done

**Pull request closed without merging**